src/strings/en-us.json

```diff
@@ -470,7 +470,7 @@
     "HeaderTranscodingProfileHelp": "Add transcoding profiles to indicate which formats should be used when transcoding is required.",
     "HeaderTunerDevices": "Tuner Devices",
     "HeaderTuners": "Tuners",
-    "HeaderTypeImageFetchers": "{0} Image Fetchers",
+    "HeaderTypeImageFetchers": "Image fetchers ({0})",
```

Comment:
Current state:
<img src="https://user-images.githubusercontent.com/56478732/140622940-8ff35c49-bf79-4421-bfd5-b32e09d542ca.png" width="50%">
<img src="https://user-images.githubusercontent.com/56478732/140622942-fb50302e-f68d-467d-b6bd-aa9af1378e24.png" width="50%">
I'm not sure about the lack of `:` for image fetchers - it is probably because of button.
@WWWesten don't hurry, let other team members to look.

Reply:
To be honest, I don't really like those endless colons that don't make much sense. Moreover, there is no single standard on different Jellyfin pages, as, for example, if we compare it with the User page. In general, I think that colons are superfluous in front of the checkbox column, especially in front of the button. It makes more sense before the dropdown. But this is a more global task of the interface style and is not the purpose of this commit. And I cannot decide this alone.